webview/src/plots/components/styles.module.scss

```diff
@@ -343,6 +288,12 @@ $resizer-border: 2px dashed $accent-color;
   }
 }
 
+.nbItemsPerRowSlider {
+  display: flex;
+  justify-content: end;
+  padding: 0 20px;
```

Comment:
[C] When plots are large and I am at the bottom of the section then there is a lot of scrolling involved to get back up to the top if I want to adjust the size. Although this is less important if we don't think people are using it I still think is bad UX:

https://user-images.githubusercontent.com/37993418/223609270-3d6190cc-9ec4-4977-a3ca-a85cceff3b02.mov

Could make the control sticky in a follow-up? Up to you.

Reply:
Yes was thinking the same. I'll add this to the ticket, I think this is an important follow up.